StitchCoreServicesAwsSes-iOS/StitchCoreServicesAwsSes-iOSTests/StitchCoreServicesAwsSes_iOSTests.swift

```diff
@@ -0,0 +1,103 @@
+import XCTest
+
+import StitchCore
+import StitchCoreAdminClient
+import StitchCoreTestUtils_iOS
+import StitchCoreServicesAwsSes
+@testable import StitchCoreServicesAwsSes_iOS
+
+class StitchCoreServicesAwsSes_iOSTests: BaseStitchIntTestCocoaTouch {
+    private let awsAccessKeyIdProp = "test.stitch.accessKeyId"
+    private let awsSecretAccessKeyProp = "test.stitch.secretAccessKey"
+    
+    private lazy var pList: [String: Any]? = {
```

Comment:
[minor] at this point, we should add a helper function for this in `BaseStitchIntTestCocoaTouch` and generecize it:

```swift
class BaseStitchIntTestCocoaTouch {
    func fetchPlist<T: BaseStitchIntTestCocoaTouch>(_ this: T.Type) -> [String: Any]? {
        let testBundle = Bundle(for: this)
        guard let url = testBundle.url(forResource: "Info", withExtension: "plist"),
            let myDict = NSDictionary(contentsOf: url) as? [String:Any] else {
                return nil
        }
        
        return myDict
    }
}

class StitchCoreServicesAwsSes_iOSTests {
    private lazy var pList: [String: Any]? = fetchPList(Self.self)
}
```

Reply:
👍 Though the invocation that works seems to be `private lazy var pList: [String: Any]? = fetchPlist(type(of: self))` instead of using `Self.self.`